config.h

```diff
@@ -80,7 +80,6 @@
 #define HOMING_CYCLE_0 (1<<Z_AXIS)                // REQUIRED: First move 
```

Comment:
They are not used by the python client. However if you issued a raw $H on the serial interface they would be used. 
I will change to X,Y,Z+C, instead.